climada/hazard/tc_tracks.py

```diff
@@ -488,6 +492,16 @@ def read_ibtracs_netcdf(self, provider=None, rescale_windspeeds=True, storm_id=N
                            'have been found: %s%s', len(invalid_sids), ", ".join(invalid_sids[:5]),
                            ", ..." if len(invalid_sids) > 5  else ".")
             ibtracs_ds = ibtracs_ds.sel(storm=valid_storms_mask)
+            
+        if discard_single_points:
+            valid_storms_mask = ibtracs_ds.valid_t.sum(axis=1) > 1
```

Comment:
Thanks for consulting me about this change! Since single-point events are quite rare (~0.5% and mainly old tracks that never reached inhabited areas) when using `estimate_missing=True` and the default `provider`, I think it's safe to discard these events by default. Users who choose to take different parameters (e.g. different `provider`) will probably consider setting `discard_single_points=False` according to their needs.
 
Use dimension names for clarity:
```suggestion
            valid_storms_mask = ibtracs_ds.valid_t.sum(dim="date_time") > 1
```